src/fragments/widgetOptions.js

```diff
@@ -46,5 +46,7 @@ export const widgetOptionsFragemnt = graphql`
     required
     defaultZoom
     galaxyImg
+    questionId
+    saveAnswer
```

Comment:
And you won't need to add `saveAnswer` here since you can use `questionId` as your flag.